<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>add_dialog</class>
 <widget class="QDialog" name="add_dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>236</width>
    <height>241</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>232</width>
    <height>203</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="group_box">
     <property name="title">
      <string>Add item</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="name_label">
        <property name="text">
         <string>Name</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="name"/>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="type">
        <property name="minimumSize">
         <size>
          <width>125</width>
          <height>0</height>
         </size>
        </property>
        <property name="currentText">
         <string>Breakfast</string>
        </property>
        <property name="placeholderText">
         <string>Select</string>
        </property>
        <item>
         <property name="text">
          <string>Breakfast</string>
         </property>
         <property name="icon">
          <iconset resource="../assets/icon_resource.qrc">
           <normaloff>:/Users/kevin/PycharmProjects/QuickBite/gui/assets/icons/icons/filter_breakfast.svg</normaloff>:/Users/kevin/PycharmProjects/QuickBite/gui/assets/icons/icons/filter_breakfast.svg</iconset>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Lunch</string>
         </property>
         <property name="icon">
          <iconset resource="../assets/icon_resource.qrc">
           <normaloff>:/Users/kevin/PycharmProjects/QuickBite/gui/assets/icons/icons/filter_lunch.svg</normaloff>:/Users/kevin/PycharmProjects/QuickBite/gui/assets/icons/icons/filter_lunch.svg</iconset>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Snacks</string>
         </property>
         <property name="icon">
          <iconset resource="../assets/icon_resource.qrc">
           <normaloff>:/Users/kevin/PycharmProjects/QuickBite/gui/assets/icons/icons/filter_snacks.svg</normaloff>:/Users/kevin/PycharmProjects/QuickBite/gui/assets/icons/icons/filter_snacks.svg</iconset>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Soft Drinks</string>
         </property>
         <property name="icon">
          <iconset resource="../assets/icon_resource.qrc">
           <normaloff>:/Users/kevin/PycharmProjects/QuickBite/gui/assets/icons/icons/filter_softdrinks.svg</normaloff>:/Users/kevin/PycharmProjects/QuickBite/gui/assets/icons/icons/filter_softdrinks.svg</iconset>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Beverages</string>
         </property>
         <property name="icon">
          <iconset resource="../assets/icon_resource.qrc">
           <normaloff>:/Users/kevin/PycharmProjects/QuickBite/gui/assets/icons/icons/filter_beverages.svg</normaloff>:/Users/kevin/PycharmProjects/QuickBite/gui/assets/icons/icons/filter_beverages.svg</iconset>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Ice Cream</string>
         </property>
         <property name="icon">
          <iconset resource="../assets/icon_resource.qrc">
           <normaloff>:/Users/kevin/PycharmProjects/QuickBite/gui/assets/icons/icons/filter_icecream.svg</normaloff>:/Users/kevin/PycharmProjects/QuickBite/gui/assets/icons/icons/filter_icecream.svg</iconset>
         </property>
        </item>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="price_label">
        <property name="text">
         <string>Price</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QSpinBox" name="price">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>116</width>
          <height>0</height>
         </size>
        </property>
        <property name="suffix">
         <string> /-</string>
        </property>
        <property name="prefix">
         <string>Rs. </string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="quantity_label">
        <property name="text">
         <string>Quantity</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QSpinBox" name="quantity">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>116</width>
          <height>0</height>
         </size>
        </property>
        <property name="suffix">
         <string> serves</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="available_label">
        <property name="text">
         <string>Available</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QCheckBox" name="available">
        <property name="text">
         <string/>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="type_label">
        <property name="text">
         <string>Type</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="button_box">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../assets/icon_resource.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>button_box</sender>
   <signal>accepted()</signal>
   <receiver>add_dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>119</x>
     <y>182</y>
    </hint>
    <hint type="destinationlabel">
     <x>119</x>
     <y>101</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button_box</sender>
   <signal>rejected()</signal>
   <receiver>add_dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>119</x>
     <y>182</y>
    </hint>
    <hint type="destinationlabel">
     <x>119</x>
     <y>101</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
